src/main/java/com/epam/training/array/impl/ArrayLogic.java

```diff
@@ -0,0 +1,159 @@
+            }
+        }
+        return noIdenticalThreeDigitNumbersArray;
+    }
+
+    public Array getAllFibonacciNumbersFromArray(Array array) {
+        ArraySorter bubbleSorter = new BubbleArraySorter();
+        Array sortedArray = bubbleSorter.sort(array);
+
+        Array arrayWithFibonacciNumbers = new Array(new int[0]);
+
+        if (sortedArray.getItems()[2] == sortedArray.getItems()[0] + sortedArray.getItems()[1]) {
+            arrayWithFibonacciNumbers = arrayLogic.append(arrayWithFibonacciNumbers, sortedArray.getItems()[0]);
+            arrayWithFibonacciNumbers = arrayLogic.append(arrayWithFibonacciNumbers, sortedArray.getItems()[1]);
+
+        }
+
+        for (int i = 2; i < sortedArray.getItems().length; i++) {
+            if (sortedArray.getItems()[i] == sortedArray.getItems()[i - 1] + sortedArray.getItems()[i - 2]) {
+                arrayWithFibonacciNumbers = arrayLogic.append(arrayWithFibonacciNumbers, sortedArray.getItems()[i]);
+            }
+        }
+
+        return arrayWithFibonacciNumbers;
+    }
+
+    public Array append(Array array, int value) {
+        int[] newArray = Arrays.copyOf(array.getItems(), array.getItems().length + 1);
+
+        newArray[newArray.length - 1] = value;
+
+        return new Array(newArray);
+    }
+
+    public int[] generateRandomArray(int quantityOfElements, int minValue, int maxValue) {
+        Random random = new Random();
+        int[] arrayWithRandomValues = new int[quantityOfElements];
+        for (int i = 0; i < quantityOfElements; i++) {
+            arrayWithRandomValues[i] = random.nextInt(maxValue - minValue) + minValue;
+        }
+        return arrayWithRandomValues;
+    }
+
+    public int[] readArrayFromFile(File file) {
+        FileReader fileReader = null;
+        try {
+            fileReader = new FileReader(file);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+        Array arrayFromFile = new Array(new int[0]);
+
+        try (Scanner scanner = new Scanner(fileReader)) {
+            while (scanner.hasNext()) {
+                arrayFromFile = arrayLogic.append(arrayFromFile, scanner.nextInt());
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        return arrayFromFile.getItems();
+    }
+
+    public int[] readArrayFromConsole(Scanner scanner) {
+        Array arrayFromConsole = new Array(new int[0]);
+        while (scanner.hasNext()) {
+            arrayFromConsole = arrayLogic.append(arrayFromConsole, scanner.nextInt());
+        }
+        return arrayFromConsole.getItems();
+    }
+
+    private boolean isPrime(int element) {
+        if (element <= 1) {
+            return false;
+        } else {
+            for (int i = 2; i < element; i++) {
+                if (element % i == 0) {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+}
```

Comment:
@AIniuseva Could you add new line please

Reply:
@NikitaUstsiushenka Done. Please, see the change here: https://github.com/AIniuseva/array-task/pull/1/files#diff-89041de0ffea49e725515821fc3df9ecd94ca09d3fc9da675f9ae8f9e6fc77ddR158